### PR TITLE
038 add linting check

### PR DESCRIPTION
**Pull request by TaylorJCrawford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+[Discord](https://eslint.org/chat) |
+[Mastodon](https://fosstodon.org/@eslint)
+
+ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code. In many ways, it is similar to JSLint and JSHint with a few exceptions:
+
+* ESLint uses [Espree](https://github.com/eslint/espree) for JavaScript parsing.
+* ESLint uses an AST to evaluate patterns in code.
+* ESLint is completely pluggable, every single rule is a plugin and you can add more at runtime.
+
+## Run Locally
```
Comment (glenn-hamilton) [inline README.md]: Can you also mention how to potentially automatically fix any issues

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "eslint:recommended"
+    ],
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+    "rules" : {
+        "indent": ["error", 4],
+        "quotes": ["error", "double"],
```
Comment (glenn-hamilton) [inline .eslintrc]: I would prefer is this used single quotes instead of double quotes

File: README.md (hunk 1 above)
Comment (TaylorJCrawford) [reply README.md]: Fixed!

File: .eslintrc (hunk 2 above)
Comment (TaylorJCrawford) [reply .eslintrc]: Fixed!

File: services/ConnectionService.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
-import axiosInstance from "../config";
+import axiosInstance from '../config';
 
 export default class ConnectionService {
     async testConnection(): Promise<string> {
         try {
             const response = await axiosInstance.get('/api/database-active');
             const message: string = response.data;
-            console.log("message-> ", message);
+            console.log('message-> ', message);
```
Comment (glenn-hamilton) [inline services/ConnectionService.ts]: Should this line not have been reviewed?

Comment (TaylorJCrawford) [reply services/ConnectionService.ts]: Fixed!

**Review by glenn-hamilton — APPROVED** [review]

**Review by miagreen23 — APPROVED** [review]

**Pull request merged**